lib/achievements.coffee

```diff
@@ -9,6 +9,8 @@ module.exports =
       else
         new Achiever()
 
+    console.info "hellooooo"
```

Comment:
All good. Want to delete it and add your name to the contributors list? After that I'll :ship: it.